aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -87,6 +88,8 @@
   private FileIO fileIO;
   private LockManager lockManager;
   private CloseableGroup closeableGroup;
+  private boolean lakeFormationEnabled;
```

Comment:
we can remove this variable now, because in `newTableOps` we can directly call `awsProperties.glueLakeFormationEnabled()`